### PR TITLE
thursday morning update

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Can you look over what I did and make sure this looks ok?

### CODE REVIEW

File: gage_outages/outage_maps.R [hunk 1]
```diff
@@ -39,30 +39,53 @@ qpf <- readRDS("qpf.rds")
...
-# sites_with_NWIS <- sites %>%
-#   select(site_no = siteID, GAGE = `GAGE Site`) %>%
-#   left_join(siteInfo, by = "site_no")
-# 
-# fwrite(sites_with_NWIS, "sites_NWIS.csv")
+sites <- fread("filtered_sites_thurs.csv", colClasses = "character")
+sites <- filter(sites, nchar(siteID) > 0)
 
-sbtools::item_file_download("5bcf61cde4b0b3fc5cde1742", overwrite_file = TRUE,
-                            names = "sites_NWIS.csv", destinations = "sites_NWIS.csv")
```
Comment (dblodgett-usgs) [inline gage_outages/outage_maps.R:65]: Didn't want to overwrite this till I know I did the right thing!

File: gage_outages/outage_maps.R [hunk 2]
```diff
@@ -228,18 +251,20 @@ qpf <- sf::st_intersection(qpf, sf::st_buffer(sf::st_as_sf(conus), 0))
...
 # Plot it up
 ################################
-
+states.out_sf <- sf::st_as_sf(states.out)
 # Shapes by site type:
 gsMap <- ggplot() +
-  geom_polygon(aes(x = long, y = lat, group = group),
-               data = states.out, fill = "grey90",
-               alpha = 0.9, color = "grey") +
+  # geom_polygon(aes(x = long, y = lat, group = group),
```
Comment (dblodgett-usgs) [inline gage_outages/outage_maps.R:257]: Trying to get Colorado to plot underneath everything else. No dice.

Comment (ldecicco-USGS) [reply gage_outages/outage_maps.R:257]: so you prefer the geom_sf?

Comment (dblodgett-usgs) [reply gage_outages/outage_maps.R:257]: I thought maybe using sf would fix the Colorado issue.

**Pull request merged**